roles/servicetelemetry/tasks/component_events_smartgateway.yml

```diff
@@ -11,64 +11,14 @@
     elastic_pass: "{{ elasticsearch_es_elastic_user | json_query('resources[].data.elastic') | b64decode }}"
   no_log: true
 
-- name: Create default Events Smart Gateway manifest for collectd
-  set_fact:
-    smartgateway_collectd_events_manifest: |
-      apiVersion: smartgateway.infra.watch/v2alpha1
-      kind: SmartGateway
-      metadata:
-        name: '{{ meta.name }}-collectd-notification'
-        namespace: '{{ meta.namespace }}'
-      spec:
-        size: 1
-        amqpUrl: '{{ meta.name }}-interconnect.{{ meta.namespace }}.svc.cluster.local:5672/collectd/notify'
-        amqpDataSource: 'collectd'
-        serviceType: 'events'
-        debug: false
-        elasticUrl: 'https://elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local:9200'
-        useBasicAuth: true
-        elasticUser: '{{ elastic_user }}'
-        elasticPass: '{{ elastic_pass }}'
-        useTls: true
-        tlsClientCert: /config/certs/tls.crt
-        tlsClientKey: /config/certs/tls.key
-        tlsCaCert: /config/certs/ca.crt
-        tlsServerName: 'elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local'
-        resetIndex: false
-  when: smartgateway_collectd_events_manifest is not defined
-
-- name: Create default Events Smart Gateway manifest for ceilometer
-  set_fact:
-    smartgateway_ceilometer_events_manifest: |
-      apiVersion: smartgateway.infra.watch/v2alpha1
-      kind: SmartGateway
-      metadata:
-        name: '{{ meta.name }}-ceilometer-notification'
-        namespace: '{{ meta.namespace }}'
-      spec:
-        size: 1
-        amqpUrl: '{{ meta.name }}-interconnect.{{ meta.namespace }}.svc.cluster.local:5672/ceilometer/event.sample'
-        amqpDataSource: 'ceilometer'
-        serviceType: 'events'
-        debug: false
-        elasticUrl: 'https://elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local:9200'
-        useBasicAuth: true
-        elasticUser: '{{ elastic_user }}'
-        elasticPass: '{{ elastic_pass }}'
-        useTls: true
-        tlsClientCert: /config/certs/tls.crt
-        tlsClientKey: /config/certs/tls.key
-        tlsCaCert: /config/certs/ca.crt
-        tlsServerName: 'elasticsearch-es-http.{{ meta.namespace }}.svc.cluster.local'
-        resetIndex: false
-  when: smartgateway_ceilometer_events_manifest is not defined
-
-- name: Deploy instance of Smart Gateway for Collectd Events
-  k8s:
-    definition:
-      '{{ smartgateway_collectd_events_manifest }}'
-
-- name: Deploy instance of Smart Gateway for Ceilometer Events
-  k8s:
-    definition:
-      '{{ smartgateway_ceilometer_events_manifest }}'
+- name: Deploy Events Smart Gateway instance for each agent
+  vars:
+    data_type: 'events'
```

Comment:
what about prefixing these with `_` or similar so that we make sure we don't conflict with someone passing in `dataType` or something from the CRD?

Reply:
You mean having it as _data_type etc.? Sure can do.

Reply:
@paramite yea the idea basically just being that this has a local scope, not a global/user facing scope.

Reply:
Actually disregard. I tried making the suggestion, and it makes everything look super awkward. We can just document maybe the internal variables or something.